maestrowf/datastructures/yamlspecification.py

```diff
@@ -131,18 +134,18 @@ def load_specification_from_stream(cls, stream):
             logger.warning(
                 "*** PyYAML is using an unsafe version with a known "
                 "load vulnerability. Please upgrade your installation "
-                "to a more recent version! ***")
+                "to a more recent version! ***"
+            )
             spec = yaml.load(stream)
 
         logger.debug("Loaded specification -- \n%s", spec["description"])
         specification = cls()
         specification.path = None
         specification.description = spec.pop("description", {})
-        specification.environment = spec.pop("env",
-                                             {'variables': {},
-                                              'sources': [],
-                                              'labels': {},
-                                              'dependencies': {}})
+        specification.environment = spec.pop(
```

Comment:
Ran `black` on this file. With regards to style, it's spoiled me because I usually just let it take care of everything. In this case, `black` appears to have fixed the `flake8` issues, but it's a very opinionated formatter so it has re-formatted other stuff too.

Reply:
I noticed that myself -- especially for list comprehensions and other things that can run longer than a line. It did a very newline heavy style that I didn't much care for because it was harder to read than formatting it by hand. I've been meaning to see if there's a way to dial it back to like an 8.5 or a 9 from it's "cranked to 11" preference on style.

Reply:
One way is to set the max line length to 88, in this case I used `-l 79`.